code/__HELPERS/global_lists.dm

```diff
@@ -50,19 +50,32 @@
 		GLOB.materials_list[D.id] = D
 	sortList(GLOB.materials_list, /proc/cmp_typepaths_asc)
 
-	// Keybindings
+	// Keybindings (classic)
 	for(var/KB in subtypesof(/datum/keybinding))
 		var/datum/keybinding/keybinding = KB
-		if(!initial(keybinding.key))
+		if(!initial(keybinding.hotkey_keys))

```

Comment:
Are all your keybinds initially bound to something?

Reply:
Good catch, one of them was unbound